### ISSUE TITLE
[Feature Request] Add a section featuring all discussed crates in "this week in bevy" playlist

### ISSUE THREAD

**Issue by kettei-sproutty** [rust-adventure/thisweekinbevy#15]
Hi all!

I would like to propose a section where we can stay up-to-date about crates discussed in the YouTube playlist.
I sometimes happen to need a crate that I saw in a video but don't really remember which one.

Honestly I have no really an idea to which could be the best way to organize this section, if something like:

```toml
name = "bevy_*"
latest_supported_version = "0.15"
category = []
showcase = "video url"
uri = ""
```

or something more automatized and low maintaining using directly [crates.io API (example)](https://crates.io/api/v1/crates/bevy)

I understand that this can be really extra-work and not really easy to maintain, but let me know what you think about it!

P.S. Happy new year!

**Comment by ChristopherBiscardi** [comment]
Happy New Year :)

I'm considering a few different somewhat related aspects of the site in 2025. Two that are semi-related are project pages and crate pages, both intended to "group" week-to-week updates in some way. This is aimed to fix the issue that you're talking about here, where it can be hard to follow a particular project (or crate, etc) over a period of time. There's also information that needs to be repeated every week that would be nice to house in a more central location instead of re-finding and submitting it each week. Especially since project updates can often be "just a screenshot" and while I can recognize a lot of them offhand, its still extra work to dig up related links, etc for every entry.

"Project pages" would also include additional metadata like the steam page (if there is one), website, etc. 

One interesting note is that I don't cover every release for every crate. Specifically I exclude bugfix releases (not much to say about "vX.X.0 -> vX.X.1"), so some of the crate information will be automated (like the most recent version of Bevy the crate currently supports). I believe some of this more basic information can be automated through pulling and processing [the index](https://github.com/rust-lang/crates.io-index/tree/74fa2c90e5c0bfef2f771c548d07d3abd9cf51d9/be/vy) or if needed [other ways](https://crates.io/data-access). There are reasonable rate limits and such in place and its important to not overload the registry with requests but I think we'd be well under the limit with an appropriate implementation.

automating it would also make it possible to have these omitted bugfix releases mentioned in a special section that was just a list of "releases happened" similar to the PR/issue lists.

It would also be nice to refine the structure a bit more and through this kind of automation have the appropriate release urls consistently across issues. Right now I manually manage all the crate urls for releases and such which leads to a wide range of "I fairly arbitrarily chose crates.io, docs.rs, the github repo, the github changelog, etc" urls.

So tldr; yeah I agree and plan to do some of that in the coming year.
